bofire/data_models/strategies/predictives/botorch.py

```diff
@@ -30,6 +34,7 @@ class BotorchStrategy(PredictiveStrategy):
     categorical_method: CategoricalMethodEnum = CategoricalMethodEnum.EXHAUSTIVE
     discrete_method: CategoricalMethodEnum = CategoricalMethodEnum.EXHAUSTIVE
     surrogate_specs: Optional[BotorchSurrogates] = None
+    outlier_detection_specs: Optional[List[AnyOutlierDetection]] = None
```

Comment:
We could also think about providing the sceduling already here, then we do not need to create the `detect_outliers` keyword in tell. 

My idea would be the following: 

- Create an attribute called `min_experiments_before_outlier_check` (mabye someting more handy, I am very uncreative) that specifies how many minimal experiments has to be available before outlier checking.
- Create an argument which specifies in which schedule they are checked. And then we determine if we are in schedule via the modulo operator.